src/mcts/node.cc

```diff
@@ -105,7 +105,7 @@ Move Edge::GetMove(bool as_opponent) const {
 
 std::string Edge::DebugString() const {
   std::ostringstream oss;
-  oss << "Move: " << move_.as_string() << " P:" << p_;
+  oss << "Move: " << move_.as_string() << " P: " << GetP();
```

Comment:
Maybe output both int and float? It's only used for debug, may be useful to see both.